arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -7,40 +7,48 @@
  * \brief Implementation file for video input using FFmpeg.
  */
 
-#include "ffmpeg_init.h"
-#include "ffmpeg_video_input.h"
 
-#include <vital/types/timestamp.h>
-#include <vital/exceptions/io.h>
-#include <vital/exceptions/video.h>
+
```

Comment:
Sorry if I wasn't clear before... STL headers should go *last*. (Well, pedantically, "system" headers e.g. `<unistd.h>` are *last*-last, but we don't have any of those here. N.b. [`[include.group_order]`](https://github.com/Kitware/kwiver/blob/master/doc/code-style.rst#includes-include).) IOW, everything KWIVER "proper" as currently ordered, then ranges/kwiversys (the relative order of those two is a good question, but not relevant here), then external third-party stuff, then STL, then C-portability (e.g. `<cmath>`), then "system" headers.

The rule-of-thumb, BTW, is that if A depends on B, headers from A should precede headers from B; IOW, sort in order of decreasing locality/dependence.